js/tuner.js

```diff
@@ -120,22 +120,29 @@ function Tuner(){
         sort(candidates);
     }
 
-    /*
-        Population size = 100
-        Rounds per candidate = 5
-        Max moves per round = 200
-        Theoretical fitness limit = 5 * 200 * 4 / 10 = 400
-    */
-    this.tune = function(){
+    /**
+     * @param {Object} [params]
+     * @param {number} [params.population=100] Population size
+     * @param {number} [params.rounds=5] Rounds per candidate
+     * @param {number} [params.moves=200] Max moves per round
+     */
+    this.tune = function(params){
+        var config = Object.assign({}, params, {
+            // Defaults:
+            // Theoretical fitness limit = 5 * 200 * 4 / 10 = 400
+            population: 100,
+            rounds: 5,
+            moves: 200
+        });
```

Comment:
... oh crap ...
Just realized that this means the defaults overwrite the params 😱 !